zaza/openstack/charm_tests/keystone/tests.py

```diff
@@ -434,31 +439,42 @@ def _find_keystone_v3_user(self, username, domain):
     def test_100_keystone_ldap_users(self):
         """Validate basic functionality of keystone API with ldap."""
         application_name = 'keystone-ldap'
-        config = self._get_ldap_config()
+        intended_cfg = self._get_ldap_config()
+        filtered_cfg, special_cfg = self.config_current_separate_special_keys(
```

Comment:
What is being 'filtered'.  Non 'special' keys?  There needs to be a better word to define what these things are.